ai/src/ai/common/llm_lib.py

```diff
@@ -0,0 +1,129 @@
+import re
+from os import getenv
+from typing import NamedTuple
+
+from openai import OpenAI
+from openai.types.chat import (
+  ChatCompletionMessageParam,
+)
+
+SYSTEM_INSTRUCTION_PART_1_PATH = "src/ai/prompts/mesop_overview.txt"
+# SYSTEM_INSTRUCTION_PART_2_PATH = "src/ai/prompts/mini_docs.txt"
```

Comment:
[question] What's the commented out parts? Is part 2 the more extensive documentation?

Reply:
yeah